### PR TITLE
Test clusters creation with RHOS. Test is created instead test_clusters.js

### PR DESCRIPTION
**Pull request by KseniiaCherchenko**

### CODE REVIEW

**Comment by adanin** [comment]
Kseniia, please, remove rhel credentials and do not include a private and sensitive information into public repos in future.

File: runner.py [hunk 1]
```diff
@@ -17,5 +19,5 @@
     nose.main(
         argv=[os.path.dirname(os.path.abspath(__file__))] + sys.argv[1:] +
         ["--with-poteenrunnerplugin"],
```
Comment (nikita-prikazchikov) [inline runner.py]: add "--with-credentialsparserplugin" for correct work of plugin

File: tests/components/helpers/RHCredentialsParserPlugin.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        if hasattr(options, "rh_password"):
+            RED_HAT["password"] = getattr(options, "rh_password")
+        if hasattr(options, "rh_server"):
+            RED_HAT["satellite server hostname"] = \
+                getattr(options, "rh_server")
+        if hasattr(options, "rh_activation_key"):
+            RED_HAT["activation key"] = getattr(options, "rh_activation_key")
+
+    def begin(self):
+        PoteenLogger.collect_test_result()
```
Comment (nikita-prikazchikov) [inline tests/components/helpers/RHCredentialsParserPlugin.py]: copy/paste code. Please remove

File: tests/components/helpers/RHCredentialsParserPlugin.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+from optparse import OptionGroup
+from nose.plugins import Plugin
+from engine.poteen.contextHolder import ContextHolder
+from engine.poteen.poteenLogger import PoteenLogger
+from tests.components.settings import RED_HAT
```
Comment (nikita-prikazchikov) [inline tests/components/helpers/RHCredentialsParserPlugin.py]: use relative import

File: tests/modules/test_cluster_creation.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            activationKey=RED_HAT["activation key"],
+            submit=False
+        ))
+
+    @attr(env=["fakeui"], set=["smoke", "regression", "full"])
+    def test_form_creation_with_grizzly(self):
+        PoteenLogger.add_test_case(
+            "Check network settings page")
+
+        cluster_key = "cluster"
```
Comment (nikita-prikazchikov) [inline tests/modules/test_cluster_creation.py:53]: var could be converted to local string. No further usage

File: tests/modules/test_cluster_creation.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    def setUpClass(cls):
+        super(Test_Cluster_creation, cls).setUpClass()
+        PoteenLogger.add_test_suite("Network validation")
+
+    @attr(env=["fakeui"], set=["smoke", "regression", "full"])
+    def test_form(self):
+        PoteenLogger.add_test_case(
+            "Check network settings page")
+
+        cluster_key = "cluster"
```
Comment (nikita-prikazchikov) [inline tests/modules/test_cluster_creation.py:30]: var could be converted to local string. No further usage

**Comment by nikita-prikazchikov** [comment]
Please rebase on master and clean history.

**Pull request merged**